src/app/shared/services/directions/directions.service.ts

```diff
@@ -29,6 +32,12 @@ export class DirectionsService {
     return params;
   }
 
+  private setCityFilterParams(settlement: Codeficator, params: HttpParams): HttpParams {
+    params = params.set('catottgId', settlement?.id?.toString() ?? Constants.KYIV.id.toString());
+
+    return params;
+  }
```

Comment:
this pice of code is used only ones so I don't think wee need a separate method for this, move it to the getTopDirections